### PR TITLE
feat: operational guidelines

### PR DESCRIPTION
**Pull request by anfibiacreativa**
## Purpose

* ...

## Does this introduce a breaking change?

```
[ ] Yes
[ ] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ x] Documentation content changes
[ ] Other... Please describe:
```

Operational guidelines

### CODE REVIEW

**Review by spboyer — APPROVED** [review]

File: docs/development-guidelines/operational-guidelines.md [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+
+## Project (technical lead)
+
+A project technical lead may be the template owner, or a designated team member representing the technology, tool or service to high-light with the purpose of providing technical direction, coherence, and quality across the project. 
+
+A technical lead, depending on their seniority and competence, may be the ultimate decision maker and coordinator of all technical efforts, although when templates integrate multiple tools and service, we recommend to scope per technology and have experts on each category assuming this role.
+
+## Case scenarios and specification
+
+Before the team starts coding, make sure to have designed scenarios, use-cases and specification, answering thes questions
```
Comment (kristenwomack) [inline docs/development-guidelines/operational-guidelines.md]: minor typo: these

File: docs/development-guidelines/operational-guidelines.md [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+
+- What services, tools and products are we showcasing?
```
Comment (kristenwomack) [inline docs/development-guidelines/operational-guidelines.md]: Nit: recommend sweeping through and applying the Oxford comma on the lists, like in "services, tools, and products"

File: docs/development-guidelines/operational-guidelines.md [hunk 3]
```diff
@@ -0,0 +1,99 @@
...
+- What are our technical requirements?
+- What is the use case/functional specification? 
+    * Who will interact with your application?
+    * How does the user interact with the application?
+    * What does the application take as input?
+    * What is the application output?
```
Comment (kristenwomack) [inline docs/development-guidelines/operational-guidelines.md:27]: Really like the specificity here.

File: docs/development-guidelines/operational-guidelines.md [hunk 4]
```diff
@@ -0,0 +1,99 @@
...
+- What are the non-functional specifications, apart from the perfomance, security and usability success criteria defined by the standardization effort?
+
+### Creating a backlog
+
+Once you have the use-case and specification in place, create a backlog for all features to be developed, and prioritize them. Assign issues to respective experts, and check the progress on a regular basis.
+Use the backlog to create scopes mapping to [milestones](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/about-milestones), and assign them to the corresponding milestone.
+
+Progress can be shared during standups or async on a Teams chat.
+
+When the issue is completed, close it, referencing the pull request that has addressed it. For example comment and close with `Fixed by #[number of PR]`
```
Comment (kristenwomack) [inline docs/development-guidelines/operational-guidelines.md]: Recommendation to add a reminder that it's a good practice to close with comment for folks to follow along with the context async.

File: docs/development-guidelines/operational-guidelines.md [hunk 5]
```diff
@@ -0,0 +1,99 @@
...
+
+After locally working on your application, and before publishing your template to the gallery, automatically or on demand, an automated worflow will run in your repo to validate the standardization success criteria is met.
+
+The system will open related issues when found non-conforming or non-standard blockers in your repository. Once addressed, you can run the validation tool manually, to request reconsideration.
+
+## Version control
+
+Version control ensures code integrity and taceability. Our open-source teams typically collaborate using repositories hosted on Azure-Samples or other organizations.
+
+We advise teams to losely use [GitHub](https://docs.github.com/en/get-started/using-github/github-flow) branching flows and development cycles, always branching out from default branch (default is `main`) and only integrating with a pull-request, with at least 1 reviewer. We strongly recommend you secure your default branch with as a [protected branch](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/managing-protected-branches/about-protected-branches)
```
Comment (kristenwomack) [inline docs/development-guidelines/operational-guidelines.md]: Nit: I would drop "losely" here. I think you mean "loosely" but I don't think you need it.

File: docs/development-guidelines/operational-guidelines.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+- *Commit Practices*:
+    - *Atomic commits*: Make small, atomic commits with clear messages.
+    - *Commit messages*: When possible, follow [Conventional commits](https://www.conventionalcommits.org/en/v1.0.0/) for clarity. When this is not possible, follow the convention: type(scope): description (e.g., [identity]]: add identity logic).
+
+- *Pull Requests (PRs)*:
+    - *Code reviews*: Require at least one peer review for each PR. If there are multiple services or products involved, or specific language experts, request their review.
+    - *Automated testing and validation*: Integrate automated end-to-end testing workflows to run on PR submissions. Make sure the automated validation pipeline [coming up] is in place.
+
+- *Documentation*:
+    - Make sure all the standadrization designated files like readme.md, licence.md, contributing.md, etc, are in place and follow the conventions.
```
Comment (kristenwomack) [inline docs/development-guidelines/operational-guidelines.md]: Technically `etc` needs a period `etc.`

**Review by kristenwomack — APPROVED** [review]
Looks great! Huge progress. I left a few edits and corrections.

File: docs/development-guidelines/operational-guidelines.md (hunk 1 above)
Comment (anfibiacreativa) [reply docs/development-guidelines/operational-guidelines.md]: fixed!

File: docs/development-guidelines/operational-guidelines.md (hunk 2 above)
Comment (anfibiacreativa) [reply docs/development-guidelines/operational-guidelines.md]: fixed!

File: docs/development-guidelines/operational-guidelines.md (hunk 5 above)
Comment (anfibiacreativa) [reply docs/development-guidelines/operational-guidelines.md]: Removed!

**Pull request merged**